NotificationService/NotificationProviders/DirectSend.Shared/DirectSendMailService.cs

```diff
@@ -82,7 +82,7 @@ public async Task SendEmailAsync(EmailMessage emailMessage)
             }
 
             message.From.AddRange(emailMessage.FromAddresses.Select(x => new MailboxAddress(this.mailConfiguration.FromAddressDisplayName, x.Address)));
-            message.Importance = MessageImportance.High;
+            message.Importance = (MimeKit.MessageImportance)emailMessage.Importance;
```

Comment:
See if both are matching. Did you tested this? is this conversion working?

Reply:
yes, I verified the values and made changes accordingly. earlier our priority enums were written wrongly. Low is 0, normal is 1 and High is 2 for both Graph and Direct Send.